dotnet/samples/Solutions/04-Planner/README.md

```diff
@@ -0,0 +1,83 @@
+# Using planner
+
+The `03-Chaining-Functions` console application shows the final solution to the [Planner](https://learn.microsoft.com/en-us/semantic-kernel/ai-orchestration/planner) doc article.
```

Comment:
I'm curious if we've talked about using aka.ms links for this kind of stuff, to make it easier to change things without editing the code.

Reply:
Also, the linked docs should be updated to include StepwisePlanner

Reply:
Also odd how the page title and breadcrumb are different.

![image](https://github.com/microsoft/semantic-kernel/assets/35783676/724d6d26-a658-4ef8-8289-637e5ee0e525)

Reply:
> Also, the linked docs should be updated to include StepwisePlanner

Agreed; I'll sync with you offline to create an additional PR for this.

Reply:
> Also odd how the page title and breadcrumb are different.
> 
> ![image](https://user-images.githubusercontent.com/35783676/258472160-724d6d26-a658-4ef8-8289-637e5ee0e525.png)

I'll update the article to align it better.

Reply:
Resolving this thread since there aren't necessary changes for this PR.